packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -254,6 +253,31 @@ - (void)testTransformFix {
   XCTAssertNil(error);
   XCTAssertEqual(avPlayer.volume, 0.1f);
 
+  // Set Picture In Picture
+  FLTPictureInPictureOverlayRect *pictureInPictureRect =
+      [FLTPictureInPictureOverlayRect makeWithTop:@0 left:@0 width:@300 height:@200];
+  FLTSetPictureInPictureOverlayRectMessage *setPictureInPictureOverlayRectMessage =
+      [FLTSetPictureInPictureOverlayRectMessage makeWithTextureId:textureId
+                                                             rect:pictureInPictureRect];
+  [videoPlayerPlugin setPictureInPictureOverlayRect:setPictureInPictureOverlayRectMessage
+                                              error:&error];
+  XCTAssertNil(error);
+
+  // Set Picture In Picture Start
+  FLTStartPictureInPictureMessage *startPictureInPicture =
+      [FLTStartPictureInPictureMessage makeWithTextureId:textureId];
+  XCTestExpectation *startingPiPExpectation =
+      [self expectationWithDescription:@"startingPictureInPicture"];
+  [player onListenWithArguments:nil
+                      eventSink:^(NSDictionary<NSString *, id> *event) {
+                        if ([event[@"event"] isEqualToString:@"startingPictureInPicture"]) {
+                          [startingPiPExpectation fulfill];
+                        }
+                      }];
+  [videoPlayerPlugin startPictureInPicture:startPictureInPicture error:&error];
+  XCTAssertNil(error);
+  [self waitForExpectationsWithTimeout:30.0 handler:nil];
```

Comment:
This fails when run on a device where `isPictureInPictureSupported` is false.  I believe this is why it's failing in CI running on an iPhone simulator that doesn't support it.
```
Failing tests:
	-[VideoPlayerTests testAudioControls]
	-[VideoPlayerTests testHLSControls]
	-[VideoPlayerTests testVideoControls]
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8792814146414854129/+/u/Run_plugin_tests/build_examples/stdout

How about:
```suggestion
  NSNumber *isPictureInPictureSupported = [videoPlayerPlugin isPictureInPictureSupported:&error];
  XCTAssertNil(error);
  XCTAssertNotNil(isPictureInPictureSupported);
  XCTAssertEqual(isPictureInPictureSupported.boolValue,
                 [AVPictureInPictureController isPictureInPictureSupported]);

  if (isPictureInPictureSupported.boolValue) {
    FLTStartPictureInPictureMessage *startPictureInPicture =
        [FLTStartPictureInPictureMessage makeWithTextureId:textureId];
    XCTestExpectation *startingPiPExpectation =
        [self expectationWithDescription:@"startingPictureInPicture"];
    [player onListenWithArguments:nil
                        eventSink:^(NSDictionary<NSString *, id> *event) {
                          if ([event[@"event"] isEqualToString:@"startingPictureInPicture"]) {
                            [startingPiPExpectation fulfill];
                          }
                        }];
    [videoPlayerPlugin startPictureInPicture:startPictureInPicture error:&error];
    XCTAssertNil(error);
    [self waitForExpectationsWithTimeout:30.0 handler:nil];
  }
```

Reply:
Indeed that is what we discussed previously. Fixed